src/java.base/share/classes/jdk/internal/misc/UnsafeConstants.java

```diff
@@ -112,11 +112,21 @@ private UnsafeConstants() {}
 
     static final int DATA_CACHE_LINE_FLUSH_SIZE;
 
+    /**
+     * The exact log (base 2) of the number of lines in the Scoped cache.
+     *
+     * @implNote
+     * The actual value for this field is injected by the JVM.
+     */
+
+    public static final int SCOPED_CACHE_SHIFT;
```

Comment:
I can't find this constant being used.   If added for future, maybe keep `UnsafeConstants` class and this static field package-private for now.

Reply:
It originally configured the number of lines in extent local cache but the implementation has changed. @theRealAph would be best to comment on this, it may be possible to delete it.

Reply:
Yes, it's dead. Thanks.